container/go/pkg/utils/utils.go

```diff
@@ -44,3 +49,49 @@ func ReadImage(src, format string) (v1.Image, error) {
 
 	return nil, errors.Errorf("unknown image format %q", format)
 }
+
+// TODO: REMOVE these two functions copied from Winnie's PR, refactor after her PR is merged.
+type formattedString map[string]interface{}
+
+// formateWithMap takes all variables of format {{.VAR}} in the input string `format`
+// and replaces it according to the map of parameters to values in `params`.
+func formatWithMap(format string, params formattedString) string {
+	msg := &bytes.Buffer{}
+	template.Must(template.New("").Parse(format)).Execute(msg, params)
+	return msg.String()
+}
+
+// Stamp provides the substitutions of variables inside {} using info in file pointed to
```

Comment:
Make this comment more information by giving an example and referring to the function parameters.